app/javascript/components/forms/DeletePresentationForm.jsx

```diff
@@ -12,18 +12,18 @@ import useDeletePresentation from '../../hooks/mutations/rooms/useDeletePresenta
 export default function DeletePresentationForm({ handleClose }) {
   const methods = useForm();
   const { friendlyId } = useParams();
-  const { handleDeletePresentation, isLoading } = useDeletePresentation(friendlyId);
+  const deletePresentation = useDeletePresentation(friendlyId);
```

Comment:
I think it would be better if we destruct that since we're using that pattern heavily with props.
`const { mutation, isLoading } = useDeletePresentation(...);`
It feels more natural to have `disabled={isLoading}` rather than `deletePresentation.isLoading`.